### DISCUSSION TITLE
some kind of rules for the bug tracker

### DISCUSSION THREAD

**Discussion by ghost** [studio-minus/ppg-bugs-and-requests#2198] [Ideas]
You know the web page thing where there are many letters that are super important? we need this for the bug tracker so people wont act dumb in the bug tracker when they make a bug report or feature request and in the comments too the arguments sure are entertaining but they get off topic fast and who needs that?

**Comment by dinevolence** [comment]
I think this was pretty weirdly phrased but I completely agree nonetheless. I think some rules are overdue for this place.

Though, technically we already have some rules:
1. Don't multi-request
2. Use the template
3. Don't post the same request or bug again and again
4. Be over the age of 13
5. Stay on-topic

and probably more.

**Reply by ghost** [reply]
i'm indeed weird and weirdly phrase thingies because i'm too lazy to put commas and dots and capitalise everything and i always put and's everywhere and ya `now just a bad person in general

**Comment by mestiez** [comment]
Where would these even go? There's no specific spot for them as far as I'm aware

**Reply by dinevolence** [reply]
Pinned issue or pinned discussion perhaps?

**Reply by ghost** [reply]
readme file
